extensions/sql-database-projects/src/dialogs/addDatabaseReferenceQuickpick.ts

```diff
@@ -116,16 +116,31 @@ async function addSystemDatabaseReference(project: Project): Promise<ISystemData
 
 	const selectedSystemDb = await vscode.window.showQuickPick(
 		getSystemDbOptions(project),
-		{ title: constants.systemDatabase, ignoreFocusOut: true, });
+		{ title: constants.systemDatabase, ignoreFocusOut: true });
 	if (!selectedSystemDb) {
 		// User cancelled
 		return undefined;
 	}
 
-	// 3. Prompt DB name
+	// 3 Prompt for Reference Type if it's an SDK-style project
+	let referenceType = SystemDbReferenceType.ArtifactReference;
+	if (project.sqlProjStyle === ProjectType.SdkStyle) {
+		const referenceTypeString = await vscode.window.showQuickPick(
+			[constants.packageReference, constants.artifactReference],
+			{ title: constants.referenceTypeRadioButtonsGroupTitle, ignoreFocusOut: true }
+		);
+
+		if (referenceType === undefined) { // need to check for specifically undefined here because the enum SystemDbReferenceType.ArtifactReference evaluates to 0
```

Comment:
Shouldn't this be `referenceTypeString` instead?

Reply:
yup good catch!